src/libraries/System.Text.Json/tests/Serialization/TestClasses/TestClasses.Constructor.cs

```diff
@@ -2059,6 +2059,61 @@ public void Verify()
             Assert.Equal(0, Y); // We don't set parameter default value here.
         }
     }
+    
+    public class Point_MembersHave_JsonInclude : ITestClass
+    {
+        [JsonInclude]
+        public int X { get; }
```

Comment:
Add another property that has a non-public accessor, without the attribute, to show that deserialization works regardless. Then, add a comment that says we should add another test for non-public members when https://github.com/dotnet/runtime/issues/31511 is implemented.